libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -635,10 +635,16 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
 
         &::before {
             top: 11px;
-            color: #6d7680;
+            color: #b0beca;
         }
 
-        &:hover {
+        &:hover.disabled {
+            color: #b0beca;
```

Comment:
Should we change the color code here, as you can see from the attached screenshot, `#6d7680` is used in the mockup.

Reply:
the color of image of icon model and sources is #6d7680 so if we use like mock up it;s not suitable in upload and the other (upload is used font icon and the others is used image background)

Reply:
the icon of model and source is an image , we have 2 image and cant change the color so we cant change like the mockup ( the only icon can change the color like this is upload icon because it is from font icon)